### PR TITLE
Opportunistically parse resolution/size of disconnected monitors

### PR DESCRIPTION
**Pull request by q3k**
Disconnected xrandr monitors might still have a resolution set (this
happens when you disconnect but not disable a monitor). They might also
have a size set, but I haven't seen this happen.

This modifies the output parsing code to set those if found. There
already was a test case with such a situation, so we just add additional
asserts to actually ensure this data gets parsed.

### CODE REVIEW

File: xrandr_test.go [hunk 1]
```diff
@@ -1,9 +1,10 @@
 package xrandr_test
 
 import (
+	"testing"
```
Comment (q3k) [inline xrandr_test.go:4]: Sorry, looks like gofmt/goimports did some drive-by changes here. Should I revert them?

Comment (vcraescu) [reply xrandr_test.go:4]: it's ok.
